small-multiple-bar/index.js

```diff
@@ -0,0 +1,250 @@
+
+ //    social: gChartframe.socialFrame(sharedConfig)
+ // .margin({ top: 140, left: 50, bottom: 138, right: 40 })
+ // // .title("Put headline here")
+ // .width(612)
+ // .height(612), // 700 is ideal height for Instagram
+
+    video: gChartframe.videoFrame(sharedConfig)
+ .margin({ left: 207, right: 207, bottom: 210, top: 233 })
+ // .title("Put headline here")
+     .extend('numberOfColumns', 6)
+     .extend('numberOfRows', 2),
+};
+
+
+// add the frames to the page...
+d3.selectAll('.framed')
+  .each(function addFrames() {
+      const figure = d3.select(this);
+      figure.select('svg')
+          .call(frame[figure.node().dataset.frame]);
+  });
+parseData.fromCSV(dataFile, dateStructure, { xMin, dataDivisor }).then(({ data, plotData, valueExtent }) => {
+    Object.keys(frame).forEach((frameName) => {
+        const currentFrame = frame[frameName];
+
+        // define other functions to be called
+        const myXAxis = gAxis.xLinear();// sets up yAxis
+        const myYAxis = gAxis.yOrdinal();// sets up date xAxis
+        const myLegend = gLegend.legend(); /* sets up the legend */ // eslint-disable-line
+        const plotDim = currentFrame.dimension();// useful variable to carry the current frame dimensions
+
+        // Create the plot widths, but for each individual graph
+        const heightOfSmallCharts = ((currentFrame.dimension().height / currentFrame.numberOfRows()) - (currentFrame.rem() * 2.5));
+
+        const tickSize = heightOfSmallCharts;// Used when drawing the yAxis ticks
+
+
+        // draw the chart holders
+        const chart = currentFrame.plot()
+        .selectAll('g')
+        .data(plotData)
+            .enter()
+        .append('g')
+        .attr('id', d => d.name)
+        .attr('class', 'columnHolder')
+        .attr('xPosition', (d, i) => i % currentFrame.numberOfColumns());
+
+        // draw tint highlight holders
+        const tints = chart // eslint-disable-line
+            .append('g')
+            .attr('class', 'tint');
+
+        const myTints = barChart.drawTints();
+        const myChart = barChart.draw();
+
+
+        myYAxis
+        .align(yAxisAlign)
+        .domain(data.map(d => d.name))
+        .rangeRound([0, heightOfSmallCharts])
+        .frameName(frameName)
+        .paddingInner(0.2);
+
+        // Draw the yAxis
+        chart
+            .call(myYAxis);
+
+
+        // return the value in the variable newMargin and move axis if needed
+        if (yAxisAlign === 'right') {
+            const newMargin = myYAxis.labelWidth() + currentFrame.margin().right;
+            // Use newMargin redefine the new margin and range of xAxis
+            currentFrame.margin({ right: newMargin });
+            myYAxis.yLabel()
+                .attr('transform', `translate(${currentFrame.dimension().width + myYAxis.labelWidth()},${0})`);
+        } else {
+            const newMargin = myYAxis.labelWidth() - (currentFrame.rem() * 0.7) + currentFrame.margin().left;
```

Comment:
`eslint` is still giving a `no-mixed-operators` error even though the order of operations here is fairly obvious IMO.  Maybe just to satisfy it, replace with:

```
const newMargin = (myYAxis.labelWidth() - (currentFrame.rem() * 0.7)) + currentFrame.margin().left;
```